Geometry/HGCalMapping/plugins/alpaka/HGCalMappingCellESProducer.cc

```diff
@@ -53,6 +97,8 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
         for (uint32_t i = 0; i < size; i++)
           cellParams.view()[i].valid() = false;
 
+        auto offsetMap = makeOffsetMap(offsetfile_);
```

Comment:
you could modify this method to receive `const HGCalMappingCellIndexer& cellIndexer` as an extra argument